tools/swift-inspect/Sources/SwiftInspectLinux/AuxVec.swift

```diff
@@ -0,0 +1,73 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import LinuxSystemHeaders
+
+internal class AuxVec {
+  enum Error: Swift.Error { case FileReadFailure(_ filePath: String) }
```

Comment:
Why is this part of `AuxVec`?

Reply:
eh, I have it throw that error when `load()` fails but I can just as easily replace `throws` with an optional result